client/components/RelatedProducts/RelatedProducts.jsx

```diff
@@ -47,13 +55,37 @@ const RelatedProducts = ({productId}) => {
     })
   }, [productId]);
 
+  const addToOutfit = () => {
+    if (!outfitStylesList.includes(currentStyle.style_id)) {
+      const outfitStylesListCopy = outfitStylesList.slice();
+      outfitStylesListCopy.push(currentStyle.style_id);
+      setOutfitStylesList(outfitStylesListCopy);
+
+      const outfitCardsDataCopy = outfitCardsData.slice();
+      const outfitCardData = {
+        name: currentStyle.name,
+        original_price: currentStyle.original_price,
+        sale_price: currentStyle.sale_price,
+        photo: currentStyle.photos[0].url,
+        style_id: currentStyle.style_id,
+        category: product.category,
+        rating: averageRating
+      }
+      outfitCardsDataCopy.push(outfitCardData)
+      setOutfitCardsData(outfitCardsDataCopy)
+    } else {
+      alert('This style has already been added to your outfit list!')
```

Comment:
I think the alert is fine for now, but in the future, I think you might be able to use some material-ui component to do something similar.

Reply:
ooo good idea.  Didn't even think about that.  I'll investigate and create a separate ticket for it